src/app/boot/express.js

```diff
@@ -29,7 +30,9 @@ function boot(app) {
   app.use(routes);
 
   app.use(routeNotFound);
-  app.use(errorHandler);
+  app.use((error, request, response, next) =>
+    errorHandler(error, request, response, next)
+  );
```

Comment:
Would the spread operator work in this case?

e.g:
```javascript
app.use((...params) =>
     errorHandler(...params)
);
```

Reply:
Great suggestion, reduces unnecessary repetitions. I am going to test it.

Reply:
1. No, it does not work. Using `...params` as the parameter signature for the arrow function makes our application not recognize it as a error handling middleware, so it is not called. I believe we need to explicitly receive `error`, `request`, `response` and `next` as parameters, or it will not recognize it as an error handling middleware.

Result:
![image](https://user-images.githubusercontent.com/60204789/107864787-c6a1ac80-6e2d-11eb-9661-63bb9562bd58.png)

2. But you made me think about other approaches to the problem, it turns out I can simply mount it on the App instance: 
`app.use(errorHandler)`

Result:
![image](https://user-images.githubusercontent.com/60204789/107864815-05376700-6e2e-11eb-861f-808a977af90e.png)

Reply:
Awesome!

I forgot about passing the function directly as a callback.